munigeo/importer/uusimaa.py

```
@@ -0,0 +1,381 @@
+    name = "uusimaa"
+    addresses_imported = 0
+    streets_imported = 0
+    streets_enriched_with_swedish_translation = 0
+    postal_code_areas_enriched = 0
+    postal_code_areas_added_to_addresses = 0
+    postal_code_areas_created = 0
+    duplicate_addresses = 0
+    coord_transform = CoordTransform(SOURCE_SRS, TARGET_SRS)
+
+    # Contains the streets of the current municipality, used for caching
+    streets_cache = {}
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)